### PR TITLE
Modify storyTranslationById query to return number of content lines i…

### PR DESCRIPTION
**Pull request by jennifer-lu**
…n story

## Notion ticket link

[Modify storyTranslationById query to return number of content lines in story](https://www.notion.so/uwblueprintexecs/Modify-storyTranslationById-query-to-return-number-of-content-lines-in-story-6dc883260ad246c1ad931e2575948cfa)

## Implementation description

- Return number of content lines in storyTranslationById query
- Calculated using `len(translation_contents)`

## Steps to test

1. Login as Carl Sagan
2. Go to http://localhost:3000/translation/6/13
3. Delete or add some translation lines (e.g. such that Translation Progress: 70%)
4. Send for review
5. Login as Dwight Eisenhower
6. Go to http://localhost:3000/review/6/13
7. Approve or disapprove some translation lines (e.g. such that Review Progress: 40%)
8. Login as Angela Merkel
9. Go to http://localhost:3000/story/6/13
10. Check that Translation Progress and Translation Progress are correct (e.g. 70% and 40%, respectively)
11. Repeat 1-10 with different percentages

## What should reviewers focus on?

- Does percentage work?
- Is code readable and correct?

## Checklist

- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] For backend changes, I have run the appropriate linters: `docker exec -it planet-read_py-backend_1 /bin/bash -c "black . && isort --profile black ."` and I have generated new migrations: `flask db migrate -m "<your message>"`
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: backend/python/app/services/implementations/story_service.py [hunk 1]
```diff
@@ -561,6 +565,9 @@ def _get_num_approved_lines(self, translation_contents):
 
         return count
 
+    def _get_num_content_lines(self, translation_contents):
```
Comment (gaoxk) [inline backend/python/app/services/implementations/story_service.py:568]: i think this function is not entirely necessary, feel free to just
`response["num_content_lines"] = len(response["translation_contents"])`

**Review by gaoxk — APPROVED** [review]
noiceeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee 🚀

**Review by gaoxk — APPROVED** [review]
sickkkkk

**Comment by jennifer-lu** [comment]
> sickkkkk

I'm a bit worried about the failed circleci tests. Should I look into them before merging?

**Comment by gaoxk** [comment]
ah shouldve mentioned in the dev channel, my bad. theyre known to be broken right now. merge away!!

**Pull request merged**